CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 
+## 2.1.0
```

Comment:
I've created https://github.com/Element84/filmdrop-aws-tf-modules/issues/8 to track the need to formalize our versioning practices.